packages/terra-consumer-nav/src/components/user-profile/UserProfile.jsx

```diff
@@ -50,26 +47,30 @@ const propTypes = {
 };
 
 const defaultProps = {
-  avatar: <IconPerson />,
   profileLinks: [],
-  isSignIn: false,
   id: 'terra-conumser-nav-profile-button',
 };
 
 const UserProfile = ({
-  userName, avatar, id, signoutUrl, signinUrl, isSignIn, profileLinks, handleClick, intl, ...customProps
+  userName, avatar, id, signoutUrl, signinUrl, profileLinks, handleClick, intl, ...customProps
 }) => {
   let profileContent;
-  if (isSignIn) {
-    profileContent = (
-      <button className={cx('popup-button')} href={signinUrl}>
-        <Arrange
-          fitStart={<div className={cx('avatar')}>{avatar}</div>}
-          fill={<FormattedMessage id="Terra.Consumer.UserProfile.signin" />}
-          align="center"
-        />
-      </button>
-    );
+  const avatarIcon = <div className={cx('avatar')}>{avatar || <IconPerson />}</div>;
+
+  const setStaticProfile = (url, translationId) => (
+    <Button className={cx('popup-button')} href={url}>
```

Comment:
don't pass the class to the terra button. It causes bugs (see here https://github.com/cerner/terra-consumer/pull/51 ). do you need terra button since we are overriding all of its look?

Reply:
i tried just using an a tag here, but the styles of the 'popup-button' were super glitchy. I could use a regular button and have a function to set the window.location but that seems like a last resort. Are there other options I could try here?

Reply:
I'll try the same technique used in the mentioned pr

Reply:
can you try what i did in my other PR. put our class on a wrapping div that handles any position based things, remove alot of the background, border, etc stuff that using a button variant of link fixes. If there is an abosulte need for some override of terra button styles, @mhemesath said we could use important sparingly